sidebars.js

```diff
@@ -95,6 +92,7 @@ module.exports = {
       },
 
       'technical-reference/documentation-contributions',
+      'technical-reference/development-roadmap',
```

Comment:
I have moved out the roadmap page from the "code contributions" section since it is relevant for all contributors